snippets/country-localization.liquid

```diff
@@ -13,10 +13,12 @@
     aria-controls="{{ localPosition }}List"
     aria-describedby="{{ localPosition }}Label"
   >
-    <span>
-      {{- localization.country.currency.iso_code }}
-      {{ localization.country.currency.symbol }} | {{ localization.country.name -}}
-    </span>
+    
+    <span  class="header_localization-form-wrapper">
```

Comment:
I think here it's a snippet which is reused also in the footer so applying this `header` specific class seem off. If it's something coming up everywhere, it can probably be a more generic class name. Something like: `localization-form__country-wrapper` maybe 🤔